src/sap/fhir/model/r4/lib/FHIRRequestor.js

```diff
@@ -50,6 +50,7 @@ sap.ui.define([
 			rHash : /#/g,
 			rPlus : /\+/g
 		};
+		this.bSecureSearch = oModel && oModel.bSecureSearch ? oModel.bSecureSearch : false;
```

Comment:
if we default in the model class, there should be no additional defaulting necessary

Reply:
done